src/python/pants/notes/2.7.x.md

```diff
@@ -1,3 +1,43 @@
+# 2.7.x Stable Releases
+
+## 2.7.0.dev3 (Aug 06, 2021)
+
+### New Features
+
+* Add `skip_tests` field to `python_tests` to facilitate incremental migrations ([#12510](https://github.com/pantsbuild/pants/pull/12510))
+
+* Add experimental lockfile support to IPython, Pytest, and Pylint ([#12491](https://github.com/pantsbuild/pants/pull/12491))
```

Comment:
I've generally been not documenting these because it's so experimental. Once Poetry lands, I think I'd be comfortable announcing.